data/defaults.yaml

```diff
@@ -269,6 +269,12 @@ ora_profile::database::patch_levels:
         db_sub_patches:        ['34765931','34768559']
         grid_sub_patches:      ['34765931','34768559','34768569','34863894','33575402']
         required_opversion:    12.2.0.1.34
+    APR2023RU:
+      "35037840-GIRU-19.19.0.0.230418":
+        file:                  "p35037840_190000_Linux-x86-64.zip"
+        db_sub_patches:        ['35042068','35050331']
+        grid_sub_patches:      ['35050325','35107512','33575402']
```

Comment:
The db_sub_patches need to be included into the grid_sub_patches as well, same as for all the other PSU's

Reply:
Oh yeah I could do that.

Regarding the ojvm patches, we don't use these, so I didn't add them.
I could add them, but then it would be untested, but just based on oracle documentation. If that is fine I will add them